src/cmd/services/m3coordinator/ingest/write.go

```diff
@@ -572,12 +580,12 @@ func (d *downsamplerAndWriter) Storage() storage.Storage {
 	return d.store
 }
 
-func (d *downsamplerAndWriter) reportWrittenBySource(source ts.SourceType) {
-	written, ok := d.metrics.writtenBySource[source]
+func reportBySource(metrics bySource, source ts.SourceType) {
```

Comment:
nit: maybe make this a method on `bySource`?

Reply:
Oh, good point, will change.